notebooks/01_prepare_data/wikidata.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>in the other notebooks we put the imports on the first cell</p>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/778/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Implemented here: 9c03398